astropy/modeling/core.py

```diff
@@ -1074,22 +1074,28 @@ def evaluate(self, *args, **kwargs):
 
     def render(self, out=None, coords=None):
         """
-        Evaluates a model on an input array. Evaluation is limited to
-        a bounding box if the `Model.bounding_box` attribute is set.
+        Evaluates a model on an input array.  I.e., "discretizes" the model.
```

Comment:
Just to clarify, it evaluates it at fixed positions, but doesn't evaluate it over the whole pixel like discretize_model in http://docs.astropy.org/en/stable/api/astropy.convolution.discretize_model.html#astropy.convolution.discretize_model right? I wonder if we need to make that clearer.